std/array.d

```diff
@@ -3657,6 +3657,26 @@ if (isDynamicArray!A)
     static assert(!__traits(compiles, () pure { test!false(); }));
 }
 
+@system unittest // issue 19572
+{
+    static struct Struct
+    {
+        int value;
+
+        int fun() const { return 23; }
+
+        alias fun this;
+    }
+
+    Appender!(Struct[]) appender;
+
+    appender.put(const(Struct)(42));
+
+    auto result = appender.data[0];
+
+    assert(result.value != 23);
```

Comment:
Apparently this fails.

Reply:
Correct, since it depends on the ongoing DMD PR.

Reply:
Oh whoops. That needs a changelog but otherwise looks good.

Reply:
Changelog will be automatically generated as it mentions an issue

Reply:
No, without a changelog it gets a generic `issue NNNNN was fixed` entry. A specific changelog entry is for highlighting things that may be of interest, e.g. w.r.t changing behaviour such as this.

Reply:
(I was talking about this specific PR here which doesn't need a detailed changelog.)